# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/*/slim/no_slimlock_if_no_PAM.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- slim-1.3.6/CMakeLists.txt.vanilla	2020-09-30 21:27:00.614767770 +0200
+++ slim-1.3.6/CMakeLists.txt	2020-09-30 21:28:19.125780753 +0200
@@ -221,7 +221,9 @@
 ####### install
 # slim
 install(TARGETS slim RUNTIME DESTINATION bin)
+if (BUILD_SLIMLOCK)
 install(TARGETS slimlock RUNTIME DESTINATION bin)
+endif (BUILD_SLIMLOCK)
 
 if (BUILD_SHARED_LIBS)
 	set_target_properties(libslim PROPERTIES
@@ -236,7 +238,9 @@
 
 # man file
 install(FILES slim.1 DESTINATION ${MANDIR}/man1/)
+if (BUILD_SLIMLOCK)
 install(FILES slimlock.1 DESTINATION ${MANDIR}/man1/)
+endif (BUILD_SLIMLOCK)
 # configure
 install(FILES slim.conf DESTINATION ${SYSCONFDIR})
 # systemd service file
